FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -376,110 +547,280 @@ paths:
+          type: string
+          format: uuid
+      - name: reporter
+        in: query
+        description: Filter results by the reporter. Can leave null to not filter
+        schema:
+          type: string
+          format: uuid
+      - name: code
+        in: query
+        description: Filter results by fault code. Can leave null to not filter
+        schema:
+          type: integer
+          format: int32
+      - name: severity
+        in: query
+        description: Filter results by fault severity. Can leave null to not filter
+        schema:
+          type: string
+          enum:
+          - LOW
+          - MEDIUM
+          - HIGH
+          - UNKNOWN  
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ActiveFault'
+        401:
+          description: Unauthorized
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false
+      security:
+      - SystemRead:
+        - fraihmwork.system.read
+
+  /fault/{uuid}:
+    get:
+      tags:
+        - Faults
+      summary: findFaultByUuid
+      operationId: findFaultByUuid
+      parameters:
+        - name: uuid
+          in: path
+          description: uuid
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        200:
+          description: OK
           content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ActiveFault'
+        401:
+          description: Unauthorized
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden
+          content: 
             application/json:
               schema:
                 $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false      
       security:
-      - FaultRead:
-        - fraihmwork.fault.read
-    
+      - SystemRead:
+        - fraihmwork.system.read
+
     put:
       tags:
-      - Fault
+      - Faults
+      - Fault Updates
       summary: Updates a fault that is affecting a system
       description: Allows a system to report one or more faults currently affecting the system
```

Comment:
"Allows a **component** to report one or more faults currently affecting the **component**"

Reply:
Fixed and scrubbed